todo-app/src/App.js

```diff
@@ -1,10 +1,68 @@
 import React, { Component } from 'react';
+import "./style.css"
+import Header from "./components/Header.js"
+import TodoList from "./components/TodoList"
+import myTodoList from "./components/myTodoList.json"
+import moment from "moment"
+import AddingContainer from "./components/AddingContainer"
+
 
 class App extends Component {
+  state = {
+    tasks: myTodoList
+  }
+  componentWillUpdate(nextProps, nextState) {
+    localStorage.setItem("todo list", JSON.stringify(nextState.tasks))
+  }
+  componentDidMount() {
+    if (localStorage.getItem("todo list")) {
```

Comment:
As shown in class, we can actually do this inside the initial state object. Loading data from localStorage inside `componentDidMount` would hypothetically lead to a flash of different content as the data is loaded in and state is set.